data/856/327/01/85632701.geojson

```diff
@@ -527,8 +558,8 @@
     "wof:lang":[
         "eng"
     ],
-    "wof:lastmodified":1587428800,
-    "wof:name":"Arunachal Pradesh (Southern Tibet)",
+    "wof:lastmodified":1616015335,
+    "wof:name":"Arunachal Pradesh",
```

Comment:
Should the southern boundary of `Arunachal Pradesh` match up with the region boundary?

Reply:
This geometry was updated in the most recent commit, we should be good here.